docs/build/variants.md

```diff
@@ -0,0 +1,102 @@
+# Tutorial: Adding variants
+
+In this tutorial, we will show you how to use variants in order to build a pixi package against different versions of a dependency.
```

Comment:
```suggestion
In this tutorial, we will show you how to use variants in order to build a pixi package against different versions of a dependency.
Some might call this functionality, build matrix, build configurations or parameterized builds, in the conda ecosystem this is refered to as a variant.
```

I think it would be nice to add this line to trigger the search index to find this page on these words. As I believe, a `variant` is conda specific.